spring-cloud-huawei-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/ribbon/ServiceCombServerListUpdater.java

```diff
@@ -35,7 +36,14 @@ public class ServiceCombServerListUpdater implements ServerListUpdater {
 
   private UpdateAction updateAction;
 
+  private PollingServerListUpdater pollingServerListUpdater;
+
+  private static long INITIAL_DELAY_TIME = 1000;
+
+  private static long REFRESH_INITIAL_TIME = 5 * 1000;
+
   public ServiceCombServerListUpdater() {
+    pollingServerListUpdater = new PollingServerListUpdater(INITIAL_DELAY_TIME, REFRESH_INITIAL_TIME);
```

Comment:
It's better to make `PollingServerListUpdater` final

Reply:
pollingServerListUpdater isNot paramer，just init when construction method execute，is still has any problems？

Reply:
line39, this field wii never change, So it's better to make this field final.

Reply:
ok,thank you!